mkn.sh

```
@@ -0,0 +1,67 @@
+cls
+
+set -e
+
+CARGS=${CARGS:-""}
+
+clargs(){
+  # enable gpu if support is considered available
+  set -e
+  ARGS=""
+  [ -d /opt/rocm/bin ] && ARGS="res/mkn/hip_mpi"
+  [ -z "$ARGS" ] && which clang 2>&1 > /dev/null && clang -v 2>&1 | grep -q "Found CUDA" && ARGS="res/mkn/clang_cuda"
```

Shellcheck (0.10.0)

[warning] To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).

(SC2069)